tests/spec/object/spec-get.js

```diff
@@ -58,8 +58,20 @@ define(
                 expect( get(foo, 'foo.bar.baz') ).toBe(undef);
             });
 
+            it('should return undefined for undefined input objects', function() {
+              var foo = undefined;
+
+              var undef;
+              expect( get(foo, 'bar.baz')).toBe(undef);
```

Comment:
Can you add a space between the parans `) )`, to match the spacing of the others.